README.md

```diff
@@ -78,14 +80,6 @@ To compile:
     cmake ..
     make
 
-You can then run
```

Comment:
Why remove these instructions?

Reply:
There was no intent to remove anything here, which instructions have you seen removed?

Reply:
You removed this part:

>You can then run
>```
>make install
>```
>to get a cockatrice installation inside the `release` folder, or:
>``` 
>make package
>```

Reply:
I'm sorry, that was an error on my part. I'm still getting used to github, hopefully I've fixed it.